msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -996,6 +1011,13 @@ The `extensions.alternates` property is used to hide or prioritize specific in-m
 |`hide.storeOfficeAddin.assetId`| String | 64 characters | ✔️ | Specifies the AppSource asset ID of the in-market add-in to hide.|
 |`hide.customOfficeAddin`| | | | Configures how to hide an in-market add-in that isn't distributed through AppSource.|
 |`hide.customOfficeAddin.officeAddinId`|String | 64 characters | ✔️ | Specifies the ID of the in-market add-in to hide. The GUID is taken from the app manifest `id` property if the in-market add-in uses the JSON app manifest. The GUID is taken from the `<Id>` element if the in-market add-in uses the XML app manifest. |
+|`alternateIcons`| Object | | | Specifies the main icons that are used to represent the add-in on older versions of Office. This property is **required** if the Office add-in is to be installable in Office on Mac, perpetual Office licenses, and Microsoft 365 subscription versions of Office on Windows earlier than 2304 (Build 16320.00000).|
+|`alternateIcons.icon`| Object | | ✔️ | Specifies the size and URL of the image file that is used to represent the add-in. The image must be in one of the following file formats: GIF, JPG, PNG, EXIF, BMP, or TIFF. |
+|`alternateIcons.icon.size`| Number enum | | ✔️ | This property is not currently used. The icon that the "icon.url" property points to must be 64 x 64 pixels.|
```

Comment:
Can you please validate this update? The manifest schema 1.17 mentions 3 image sizes are required with possible values as 16, 20. 24, 32, 40, 48, 64, 80.

![image](https://github.com/MicrosoftDocs/msteams-docs/assets/93579821/45c96022-be7a-4c18-b934-51f600c2f997)

Reply:
This property was given the data type "extensionCommonIcon" by mistake. This will be fixed in the next version of the schema. In the meantime, the property is not being used. It would be misleading and confusing to readers to mention the sizes and requirements for the "extensionCommonIcon" type.

Reply:
Attempting to strike a balance between platform transparency and customer usability. How about these modifications:

```
|`alternateIcons.icon.size`| Number enum | | ✔️ | <br>Supported value: `64` |
|`alternateIcons.icon.url`| String | 2048 characters | ✔️ | Specifies the full, absolute URL of the image file that is used to represent the add-in. Icon image must be 64 x 64 pixels. |
|`alternateIcons.highResolutionIcon.size`| Number enum | | ✔️ | <br>Supported value: `128`.|
|`alternateIcons.highResolutionIcon.url`| String | 2048 characters | ✔️ | Specifies the full, absolute URL of the image file that is used to represent the add-in on high DPI screens. Icon image must be 128 x 128 pixels. |

Reply:
(Even though other enum values will pass validation, I think best practice is for `size` to reflect actual icon image)

Reply:
> Attempting to strike a balance between platform transparency and customer usability. How about these modifications:
> 
> ```
> |`alternateIcons.icon.size`| Number enum | | ✔️ | Size of icon image. <br>Supported value: `64` |
> |`alternateIcons.icon.url`| String | 2048 characters | ✔️ | Specifies the full, absolute URL of the image file that is used to represent the add-in. Icon image must be 64 x 64 pixels and use one of the following file formats: GIF, JPG, PNG, EXIF, BMP, TIFF. |
> |`alternateIcons.highResolutionIcon.size`| Number enum | | ✔️ | Size of high resolution icon image. <br>Supported value: `128`.|
> |`alternateIcons.highResolutionIcon.url`| String | 2048 characters | ✔️ | Specifies the full, absolute URL of the image file that is used to represent the add-in on high DPI screens. Icon image must be 128 x 128 pixels and use one of the following file formats: GIF, JPG, PNG, EXIF, BMP, TIFF. |
> ```

@erikadoyle that will work for 3 of those 4 rows. But 128 is NOT a supported value for the data type that was mistakenly asslgned to alternateIcons. I'll make your other changes, but for `size` I think we should say "This property is reserved for future use. Set the value to 16."

Reply:
Ok, let's go with this and see what folks think.